schedule/RAID1_staging.yaml

```diff
@@ -0,0 +1,40 @@
+---
+name:           RAID1
+description:    >
+  Test for RAID1.
+  Installation of RAID1 using expert partitioner.
+vars:
+  RAIDLEVEL: 1  
+  START_AFTER_TEST: RAID0
```

Comment:
This setting has to stay in the test suite settings in openQA, as scheduler should know it before test is triggered.

Reply:
fixed on ba01a8a